R/expr-meta.R

```diff
@@ -243,10 +243,10 @@ expr_meta_is_column_selection <- function(..., allow_aliasing = FALSE) {
 #' e <- pl$col("foo")
 #' e$meta$is_column()
 #'
-#' e <- pl.col("foo") * pl.col("bar")
+#' e <- pl$col("foo") * pl$col("bar")
 #' e$meta$is_column()
 #'
-#' e <- pl.col(r"^col.*\d+$")
+#' e <- pl$col("^col\\.*\\d+$")
```

Comment:
I just suspect people are more familiar with escaping in regex than with raw strings